packages/alea-frontend/components/ForceGraphExample.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box } from '@mui/material';
+import React from 'react';
+import ForceGraph2D from 'react-force-graph-2d';
+
+interface Node {
+  id: string;
+  color: string;
+}
+
+interface Link {
+  source: string;
+  target: string;
+  color: string;
+}
+
+const ForceGraphExample = ({ connections, userIdsAndActiveStatus }) => {
+  if (!connections) return null;
+  const processedConnections = new Set();
+
+  const transformedConnections = connections.flatMap(
+    ({ senderId, receiverId }) => {
+      if (
+        connections.some(
+          (conn) =>
+            conn.senderId === receiverId &&
+            conn.receiverId === senderId &&
+            !processedConnections.has(conn)
```

Comment:
To improve readability, create a boolean variable from the following and then use in the `if` condition.
` connections.some(
          (conn) =>
            conn.senderId === receiverId &&
            conn.receiverId === senderId &&
            !processedConnections.has(conn)`